src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -161,6 +161,35 @@ public partial class Slider : Control
             return instance.valueIndicatorText.Text;
         });
 
+        /// <summary>
+        /// Bindable property of CurrentValue
+        /// <remark>
+        /// Hidden API, used for NUI XAML data binding
+        /// </remark>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty CurrentValueProperty = BindableProperty.Create(nameof(CurrentValue), typeof(float), typeof(Slider), 0.0f,
```

Comment:
It need to `TwoWay` binding, CurrentValue could be changed by data source and source also could be changed by UI
```suggestion
        public static readonly BindableProperty CurrentValueProperty = BindableProperty.Create(nameof(CurrentValue), typeof(float), typeof(Slider), 0.0f, BindingMode.TwoWay
```